tls/s2n_config.c

```diff
@@ -786,7 +777,7 @@ int s2n_config_add_ticket_crypto_key(struct s2n_config *config,
     struct s2n_blob salt = { .size = 0 };
     struct s2n_blob info = { .size = 0 };
 
-    struct s2n_blob allocator;
+    struct s2n_blob allocator = {0};
     struct s2n_ticket_key *session_ticket_key;
 
     GUARD(s2n_alloc(&allocator, sizeof(struct s2n_ticket_key)));
```

Comment:
We probably need to use some kind of clean-up here, as we may exit before we reach deallocation https://github.com/awslabs/s2n/blob/master/docs/DEVELOPMENT-GUIDE.md#cleanup-on-error

Reply:
Done.